docs/dev.md

```diff
@@ -62,3 +64,22 @@ Integration tests can read the following system properties to obtain information
 | liberty.hostname | The host name of the Liberty server. |
 | liberty.http.port | The port used for client HTTP requests. |
 | liberty.https.port | The port used for client HTTP requests secured with SSL (https). |
+
+##### devc, Container Mode
+
+This is a technology preview.
+
+Start a Liberty server in a local container using the Docker file that you provide. The Docker file must copy the application .war file and the server configuration files that the application requires into the container. An alternative to the `devc` goal is to specify the `dev` goal with the `-Dcontainer` option. A sample Docker file is shown in [Building an application image](https://github.com/openliberty/ci.docker/#building-an-application-image). Note that the context of the `docker build` command used to generate the container image is the directory containing the docker file.
+
+You need to install the Docker runtime (e.g. Docker Desktop) locally to use this Maven goal. You can examine the commands used to build and run the container by viewing the console output of dev mode.
+
+When dev mode runs in a container it still provides the same features. It monitors files for changes and runs tests either automatically or on command. This mode also allows you to attach a debugger to work on your application. You can review the logs generated by your server in the Liberty directory in your project e.g. target/liberty/wlp/usr/servers/defaultServer/logs.
+
+This mode publishes the container ports 9080, 9443 and 7777 by default. If your application needs to publish other ports add them to the `dockerRunOpts` option either on the `mvn` command line or in the `pom.xml` file. E.g. `-DdockerRunOpts="-p 9081:9081"`
```

Comment:
The second sentence mentions the mvn command line and then pom.xml, but then follows with a command line example.  The ordering of these may be confusing for readers.

Reply:
Ok